jest.config.js

```diff
@@ -22,9 +22,6 @@ module.exports = {
   testMatch: [
     '<rootDir>/cardstack/**/*.test.ts',
     '<rootDir>/cardstack/**/*.test.js',
-    // Avoid testing components for now
-    '!<rootDir>/cardstack/**/*.test.jsx',
-    '!<rootDir>/cardstack/**/*.test.tsx',
   ],
```

Comment:
We need to have the components extensions in order to execute the tests, the way it's right now we are collecting coverage from the components, but not actually testing them, it's ` 107` when it should be around `127` tests
```suggestion
    '<rootDir>/cardstack/**/*.test.jsx',
    '<rootDir>/cardstack/**/*.test.tsx',
  ],
```

Reply:
Whoops good catch. I had all the component tests listed one at a time until the end and then... removed them. 🤦‍♂️